src/site/resources/index.html

```diff
@@ -0,0 +1,160 @@
+                    </div><!-- end of .col.col-right -->
+                </div><!-- end of .container -->
+            </section><!-- end of .hero-banner -->
+
+            <section class="block featuring">
+                <div class="container">
+                    <ul class="featuring-links">
+                        <li>
+                            <a href="./apidocs/index.html" title="gcloud-java docs" class="btn">
+                                Read the Docs
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/GoogleCloudPlatform/gcloud-java" title="gcloud-java on Github" class="ext-link">
+                                <img src="img/icon-link-github.svg" alt="Github icon" />
+                                Github
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/GoogleCloudPlatform/gcloud-java/issues" title="gcloud-java issues on Github" class="ext-link">
+                                <img src="img/icon-link-github.svg" alt="Github icon" />
+                                Issues
+                            </a>
+                        </li>
+                        <li>
+                            <a href="http://stackoverflow.com/questions/tagged/gcloud-java" title="gcloud on StackOverflow" class="ext-link">
+                                <img src="img/icon-link-stackoverflow.svg" alt="StackOverflow icon" />
+				StackOverflow
+                            </a>
+                        </li>
+                        <li>
+                            <a href="./project-info.html" title="Java package manager" class="ext-link">
+                                 <img src="img/icon-link-package-manager.svg" alt="Package Manager icon" />
+                                Package Information
+                            </a>
+                        </li>
+                    </ul>
+                </div><!-- end of .container -->
+            </section><!-- end of .featuring -->
+
+            <section class="block about">
+                <div class="container clearfix">
+                  <h3 class="block-title">What is it?</h3>
+
+                  <p><code>gcloud</code> is a client library for accessing Google
+		    Cloud Platform services that significantly reduces the boilerplate
+		    code you have to write. The library provides high-level API
+		    abstractions so they're easier to understand. It embraces
+		    idioms of Java, works well with the standard library, and
+		    integrates better with your codebase.
+		    All this means you spend more time creating code that matters
+		    to you.</p>
+		  
+		  <p><code>gcloud</code> is configured to access Google Cloud Platform
+		    services and authorize (OAuth 2.0) automatically on your behalf.
+		    With a quick install and a private key, you are up and ready
```

Comment:
"a private key" - I see that this is taken from other documentation (@jgeewax feedback on it would be great) but
I wonder if that is what we should say in at this "introductory section".  Consider this scenarious:
1) AE - No need for key (auto-detected).
2) Compute engine - No need for key (auto-detected).
3) Other - Use key or authenticated using gcloud sdk.

Indeed a key could be use in all scenarios but not mandatory in either case...